### PR TITLE
Cloud Native Chapter 7

### PR DESCRIPTION
**Pull request by ardalis**
## Summary

Chapter 7 files.

### CODE REVIEW

File: docs/architecture/cloud-native/observability-patterns.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+The high performance of logging tools and the tunability of verbosity should encourage developers to log as frequently as possible. Many favor a pattern of logging entry and exit from each method. This may sound like overkill but it is very infrequent that developers will wish for less logging. In fact it is not uncommon to perform deployments for the sole purpose of adding logging around a particularly problematic method. Err on the side of too much logging and not too little. 
+
+In traditional applications the log files were typically stored on the local machine. in fact, on Unix-like operating systems there is a folder structure defined, typically under `/var/log`, to hold any logs. Of course, the utility of logging to a flat file on a single machine is vastly reduced in a cloud environment. Applications producing logs may not have access to the local disk or the local disk may be highly transient as containers are shuffled around physical machines.
```
Comment (EugeneKramer) [inline docs/architecture/cloud-native/observability-patterns.md]: in fact...->In fact

Comment (EugeneKramer) [inline docs/architecture/cloud-native/observability-patterns.md]:
```suggestion
In traditional applications the log files were typically stored on the local machine. In fact, on Unix-like operating systems there is a folder structure defined, typically under `/var/log`, to hold any logs. Of course, the utility of logging to a flat file on a single machine is vastly reduced in a cloud environment. Applications producing logs may not have access to the local disk or the local disk may be highly transient as containers are shuffled around physical machines.
```

**Review by Thraka — APPROVED** [review]

**Comment by ardalis** [comment]
@mairaw should be good to merge.

**Comment by ardalis** [comment]
Still looks good to me. :shipit:

File: docs/architecture/cloud-native/observability-patterns.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+The metric gathering capabilities of the monitoring tools can also be fed manually from within the application. Business flows that are of particular interest such as new users signing up or orders being placed, may be instrumented such that they increment a counter in the central monitoring system. This unlocks the monitoring tools to not only monitor the health of the application but the health of the business.
+
+Queries can be constructed in the log aggregation tools to look for certain statistics or patterns, which can then be displayed in graphical form, on bespoke dashboards. Frequently, teams will invest in large, wall-mounted displays that rotate through the statistics related to an application. This way, it's very simple to see the problems as they occur.
```
Comment (mairaw) [inline docs/architecture/cloud-native/observability-patterns.md]: bespoke dashboards?

**Review by mairaw — APPROVED** [review]
Looks good. @ardalis make sure you approve my changes (some are highlighted in the comments) and check if you can address the rest of comments.

**Comment by ardalis** [comment]
Ok should be good to go.

**Pull request merged**